locales-pending/onboarding.ftl

```diff
@@ -13,8 +13,8 @@ onboarding-get-started-how-it-works-dialog-heading-line1 = We’re here to prote
 onboarding-get-started-how-it-works-dialog-heading-line2 = Here’s how it works:
 onboarding-get-started-how-it-works-dialog-step1-heading = Scan for exposures
 # Variables:
-#   $dataBrokerCount (number) - number of scanned data broker sites, e.g. 190
-onboarding-get-started-how-it-works-dialog-step1-content = With just a few key pieces of information, we’ll search for you in all known data breaches and { $dataBrokerCount } major data broker sites. All users get the first scan free.
+#   $dataBrokerTotalCount (number) - number of scanned data broker sites, e.g. 190
+onboarding-get-started-how-it-works-dialog-step1-content = With just a few key pieces of information, we’ll search for you in all known data breaches and { $dataBrokerTotalCount } major data broker sites. All users get the first scan free.
```

Comment:
Always use the plural form for this type of strings, as it triggers the correct UI in Pontoon.

```
onboarding-get-started-how-it-works-dialog-step1-content = {
    $dataBrokerTotalCount -> 
        *[other] With just a few key pieces of information, we’ll search for you in all known data breaches and { $dataBrokerTotalCount } major data broker sites. All users get the first scan free.
}    
```

Reply:
(technically, you could also decide to provide the correct form for one)

```
onboarding-get-started-how-it-works-dialog-step1-content = {
    $dataBrokerTotalCount -> 
         [one] With just a few key pieces of information, we’ll search for you in all known data breaches and { $dataBrokerTotalCount } major data broker site. All users get the first scan free.
        *[other] With just a few key pieces of information, we’ll search for you in all known data breaches and { $dataBrokerTotalCount } major data broker sites. All users get the first scan free.
}    
```

Reply:
> The count will be always bigger than one.

That is not the point though. We want the plural UI to show up automatically in Pontoon, and a lot of locales have more complex rules than English (and more forms).

Reply:
Yes, I understood that this is needed for variables and I added your change request. I mean I just did not add a case for `[one]` or would that be needed always as well?

Reply:
Ah, misunderstood that. `*[other]` is enough 👍🏼